README.md

```diff
@@ -147,7 +147,8 @@ For the moment, we are currently focused on the CISA KEV Database and Google Tsu
 
 | CVE ID                                                  | Implemented | Detail                                                  | Published Date |
 |:--------------------------------------------------------|:-----------:|:--------------------------------------------------------|:--------------:|
-| CVE-2024-34470                                          |      ✅      | Custom Exploit by Ostorlab: included in Agent Asteroid  |   2024-05-29   |
+| CVE-2024-2194                                          |      ✅      | Custom Exploit by Ostorlab: included in Agent Asteroid.  |   2024-03-13   |
+| CVE-2024-34470                                         |      ✅      | Custom Exploit by Ostorlab: included in Agent Asteroid.  |   2024-05-29   |
```

Comment:
These two are not in the correct order can you please change them , the order should be according to "Published date"

Reply:
fixed! thanks.